docs/deploy.md

```diff
@@ -1,26 +1,28 @@
-# Deploying and saving a model
+# Making Predictions and Deploying Models
```

Comment:
Did you mean to rename deploy.md to predictions.md (predictions.md is included in the list of docs to audit)